erpnext/erpnext_integrations/taxjar_integration.py

```diff
@@ -203,6 +199,15 @@ def set_sales_tax(doc, method):
 
 			doc.run_method("calculate_taxes_and_totals")
 
+def check_for_nexus(doc, tax_dict):
+	TAX_ACCOUNT_HEAD = frappe.db.get_value("TaxJar Configuration", doc.company, "tax_account_head")
+	# check if to_state within region if not do not calculate sales tax.
+	if not frappe.db.get_value('TaxJar Nexus', {'region_code': tax_dict["to_state"]}):
+		for tax in doc.taxes:
+			if tax.account_head == TAX_ACCOUNT_HEAD:
+				doc.taxes.remove(tax)
+		return False
+	return True
 
 def validate_tax_request(doc, tax_dict):
```

Comment:
this function should be renamed to get_sales_tax since it not just validates but also returns the sales tax

Reply:
Yeah That's right